gwpy/astro/range.py

```diff
@@ -283,3 +287,184 @@ def burst_range(psd, snr=8, energy=1e-2, fmin=100, fmax=500):
     # normalize and return
     r = units.Quantity(result / (fmax - fmin), unit=integrand.unit) ** (1/3.)
     return r.to('Mpc')
+
+
+def range_trends(hoft, stride, fftlength=None, overlap=None,
```

Comment:
I think I would rather have this function be called `range_timeseries`.

Reply:
@duncanmmacleod, makes sense to me, this is done in commit d8dcdd0ee5bb2cc233a40ea89357183b40f24e5f including updates to all references to this function.